### PR TITLE
Support .config directory

### LINKED ISSUE #179: Support .config directory

### PR DESCRIPTION
**Pull request by jannis-baum**
Close #179

### CODE REVIEW

**Comment by tuurep** [comment]
Nice, thanks

I have some doubts that the precedence is working as intended, assuming that the `~/.config/vivify/config.json` should be applied over `~/.vivify/config.json`, when I test with both it looks like the `~/.vivify/config.json` is being applied. Could you verify?

**Comment by tuurep** [comment]
Darn it. The classic "I have an AUR package installed and didn't realize" -case. Let me try again :smile:

**Comment by tuurep** [comment]
Alright I gave it a good with each supported path and works great + I like the flexibility of it

The only thing is that if I have an empty `config.json` for example in the `~/.config/vivify/` path, I'd expect it to act as a 'default config' but when I have a `~/.vivify/config.json` at the same time, that gets applied instead

What do you think about that?

**Comment by jannis-baum** [comment]
Currently, the first path that has *a valid JSON object* will be used. An empty file is not valid JSON. We can change it to try using the first path with a file no matter what. Do you think that's better?

**Comment by tuurep** [comment]
Hmm. Well, that would match my expectation of what should happen, but is that a small and easy change?

**Comment by jannis-baum** [comment]
> Hmm. Well, that would match my expectation of what should happen, but is that a small and easy change?

Yes haha: https://github.com/jannis-baum/Vivify/pull/180/commits/bc70a8cdc9ef3462410e4500cb7503fc06d7b80a. Can you verify that it works how you would expect now?

**Review by tuurep — APPROVED** [review]
Lol it's the smallest change of all time :tada: 

Yes I think it's quite perfect now, thanks!
